docs/concepts/configuration/pod-priority-preemption.md

```diff
@@ -12,8 +12,8 @@ title: Pod Priority and Preemption
 [Pods](/docs/user-guide/pods) in Kubernetes 1.8 and later can have priority. Priority
 indicates the importance of a Pod relative to other Pods. When a Pod cannot be scheduled,
 the scheduler tries to preempt (evict) lower priority Pods to make scheduling of the
-pending Pod possible. In a future Kubernetes release, priority will also affect
-out-of-resource eviction ordering on the Node.
+pending Pod possible. In Kubernetes 1.9 and later, Priority also affects scheduling
+order of pods and out-of-resource eviction ordering on the Node.
 
 **Note:** Preemption does not respect PodDisruptionBudget; see 
 [the limitations section](#poddisruptionbudget-is-not-supported) for more details.
```

Comment:
is this limitation still valid?

Reply:
okay, there is a new subsection on this topic. We can remove the note here then.

Reply:
You are right. I will delete this note.

Reply:
Done